test/specs/lazyload.spec.js

```diff
@@ -276,7 +276,7 @@ describe('LazyLoad', () => {
 
       // let `scroll` event handler done their job first
       setTimeout(() => {
-        expect(document.querySelectorAll('.test').length).to.equal(1);
+        expect(document.querySelectorAll('.test').length).to.equal(2);
```

Comment:
For some reason when pointing to one was always failing, even before my changes. I need some help here.